awscli/botocore/data/dsql/2018-05-10/completions-1.json

```diff
@@ -0,0 +1,69 @@
+{
+  "version": "1.0",
+  "resources": {
+    "Cluster": {
+      "operation": "ListClusters",
+      "resourceIdentifier": {
+        "identifier": "clusters[].identifier",
+        "arn": "clusters[].arn"
+      }
+    }
+  },
+  "operations": {
+    "DeleteCluster": {
+      "identifier": {
+        "completions": [
+          {
+            "resourceName": "Cluster",
+            "resourceIdentifier": "identifier",
+            "parameters": {}
+          }
+        ]
+      }
+    },
+    "GetCluster": {
+      "identifier": {
+        "completions": [
+          {
+            "resourceName": "Cluster",
+            "resourceIdentifier": "identifier",
+            "parameters": {}
+          }
+        ]
+      }
+    },
+    "GetVpcEndpointServiceName": {
+      "identifier": {
+        "completions": [
+          {
+            "resourceName": "Cluster",
+            "resourceIdentifier": "identifier",
+            "parameters": {}
+          }
+        ]
+      }
+    },
+    "ListTagsForResource": {
+      "resourceArn": {
+        "completions": [
+          {
+            "resourceName": "Cluster",
+            "resourceIdentifier": "arn",
+            "parameters": {}
+          }
+        ]
+      }
+    },
```

Comment:
I'm also going to remove this because there may be other modeled resources in the future that won't be included. It would be interesting if we could group resources together to reference in `operations` so that an operation could target multiple resources.